io/net/virt-net/network_virtualization.py

```diff
@@ -129,6 +129,9 @@ def setUp(self):
             "auto_failover", '*', default=None)
         if self.auto_failover not in ['0', '1']:
             self.auto_failover = '1'
+        self.vios_ip = self.params.get('vios_ip', '*', default=None)
```

Comment:
Are we going to do this only for one vios ?

Reply:
yes

Reply:
yes only one vios of active device, as the mkdev/rmdev is done only on active device

Reply:
backing device can be between 1 to 6.. so we are not doing it for all

Reply:
ok